asab/log.py

```
@@ -427,9 +427,39 @@ def __init__(self, fmt=None, datefmt=None, style='%', sd_id='sd'):
+			elif o.tzinfo is not None:
+				# The datetime object is timezone-aware but not UTC
+				# NOT WANTED, using non-UTC timestamps is not recommended
+				return o.isoformat()
+			else:
+				# The datetime object is timezone-naive -> interpret it as UTC
+				return o.isoformat() + "Z"
+
+		# If obj is not json serializable, convert it to string
+		try:
+			return str(o)
+		except Exception:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)